wkcuber/cubing.py

```diff
@@ -95,17 +96,25 @@ def create_parser() -> ArgumentParser:
 
 
 def find_source_filenames(source_path: str) -> List[str]:
-    # Find all files in a folder that have a matching file extension
-    source_files = list(
-        find_files(path.join(source_path, "*"), image_reader.readers.keys())
-    )
+    # Find all source files that have a matching file extension
+
+    input_path = Path(source_path)
+
+    if input_path.is_dir():
+        joined_path = path.join(source_path, "**")
```

Comment:
the old code only had one `*`. is this intended for cubing nested structures? what happens if I pass a folder which contains two layers? within this pattern matching logic, the layer name extraction would need to happen. otherwise, all input files (of both layers) are thrown together, arent't they? :thinking: or is the `converter` class responsible for dissecting this? then, I'm not sure whether `**` is necessary here..

Reply:
Yes, you're correct again. Initially, I wanted to add the layer config into the cubing model, but now it is handled by the `converter` class, so I can roll back to the old version.